### PR TITLE
fix testcommonstruct

### PR DESCRIPTION
**Pull request by googs1025**
#### What this PR does / why we need it:
- modify the ut framework variable name to make it clearer

#### Which issue(s) this PR fixes:
None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/scheduler/uthelper/helper.go [hunk 1]
```diff
@@ -33,30 +33,36 @@ import (
 	"volcano.sh/volcano/pkg/scheduler/util"
 )
 
-// RegisterPlugins plugins
-func RegisterPlugins(plugins map[string]framework.PluginBuilder) {
```
Comment (lowang-bh) [inline pkg/scheduler/uthelper/helper.go:37]: This is a package method.

File: pkg/scheduler/uthelper/helper.go [hunk 2]
```diff
@@ -235,9 +257,9 @@ func (test *TestCommonStruct) CheckPGStatus(caseIndex int) error {
 // CheckPipelined checks pipeline results
 func (test *TestCommonStruct) CheckPipelined(caseIndex int) error {
 	ssn := test.ssn
-	for jobID, nodes := range test.PipeLined {
-		job := ssn.Jobs[api.JobID(jobID)]
-		if job == nil {
+	for jobID, nodes := range test.ExpectPipeLined {
+		job, ok := ssn.Jobs[api.JobID(jobID)]
+		if job == nil || !ok {
```
Comment (lowang-bh) [inline pkg/scheduler/uthelper/helper.go:262]: duplicated. not ok means job is nil.

File: pkg/scheduler/uthelper/helper.go [hunk 3]
```diff
@@ -219,9 +241,9 @@ func (test *TestCommonStruct) CheckEvict(caseIndex int) error {
 // CheckPGStatus check job's podgroups status
 func (test *TestCommonStruct) CheckPGStatus(caseIndex int) error {
 	ssn := test.ssn
-	for jobID, phase := range test.Status {
-		job := ssn.Jobs[jobID]
-		if job == nil {
+	for jobID, phase := range test.ExpectStatus {
+		job, ok := ssn.Jobs[jobID]
+		if job == nil || !ok {
```
Comment (lowang-bh) [inline pkg/scheduler/uthelper/helper.go:246]: duplicated. not ok means job is nil.

File: pkg/scheduler/uthelper/helper.go [hunk 4]
```diff
@@ -163,25 +181,29 @@ func (test *TestCommonStruct) CheckBind(caseIndex int) error {
...
+	if len(test.ExpectBindMap) != len(binds) {
+		return fmt.Errorf("case %d(%s) check bind: \nwant: %v\n got %v ", caseIndex, test.Name, test.ExpectBindMap, binds)
 	}
-	for key, value := range test.BindMap {
-		got := binds[key]
+	for key, value := range test.ExpectBindMap {
+		got, ok := binds[key]
+		errorMsg := fmt.Sprintf("case %d (%s) check bind:\nwant: %v->%v\ngot: %v", caseIndex, test.Name, key, value, key)
+		if !ok {
+			return fmt.Errorf("%s->nil", errorMsg)
```
Comment (lowang-bh) [inline pkg/scheduler/uthelper/helper.go:191]: It is not nil, just a empty string.

File: pkg/scheduler/uthelper/helper.go (hunk 3 above)
Comment (googs1025) [reply pkg/scheduler/uthelper/helper.go]: done

File: pkg/scheduler/uthelper/helper.go (hunk 2 above)
Comment (googs1025) [reply pkg/scheduler/uthelper/helper.go]: done

File: pkg/scheduler/uthelper/helper.go (hunk 1 above)
Comment (googs1025) [reply pkg/scheduler/uthelper/helper.go]: done

File: pkg/scheduler/uthelper/helper.go (hunk 4 above)
Comment (googs1025) [reply pkg/scheduler/uthelper/helper.go]: already rolled back. done

**Review by lowang-bh — APPROVED** [review]

**Pull request merged**